tests/gocase/unit/quit_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package unit
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/stretchr/testify/require"
+)
+
+func formatCommand(args []string) string {
+	suffix := "\r\n"
+	str := "*" + fmt.Sprintf("%d", len(args)) + suffix
+	for _, arg := range args {
+		str += "$" + fmt.Sprintf("%d", len(arg)) + suffix
+		str += arg + suffix
+	}
+	return str
+}
+
+func TestQuit(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{})
+	defer srv.Close()
+	rdb := srv.NewTCPClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+
+	t.Run("QUIT returns OK", func(t *testing.T) {
+		// reconnect
+		require.NoError(t, rdb.Close())
+		rdb = srv.NewTCPClient()
+		require.NoError(t, rdb.Write(formatCommand([]string{"quit"})))
+		require.NoError(t, rdb.Write(formatCommand([]string{"ping"})))
```

Comment:
In the original TCL code, I think it is not a raw TCP client here. Is there any reason for this inconsistency?